### PR TITLE
Aden 3179 move slot sizes to project43

### PR DESCRIPTION
**Pull request by shagupta92042**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java [hunk 1]
```diff
@@ -1030,65 +1030,65 @@ private AdsDataProvider() {
...
             "",
             ImmutableMap.<String, Object>builder()
                 .put("slotName", AdsContent.FLOATING_MEDREC)
                 .put("slotSize", new Dimension(300, 600))
-                .put("lineItemId", 223968492)
+                .put("lineItemId", 270230292)
                 .put("src", "gpt")
                 .build()
-        },
+        }/*,
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: MINOR Move this trailing comment on the previous empty line. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ATrailingCommentCheck)

File: src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java [hunk 2]
```diff
@@ -1030,65 +1030,65 @@ private AdsDataProvider() {
...
         {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Define a constant instead of duplicating this literal "SyntheticTests/Slots/Size/120x600" 3 times.](https://github.com/Wikia/selenium-tests/blob/6b4ce621a266ed43c5ce37a4317509ba797b1595/src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java#L1143) [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)

**Comment by ludwikkazmierczak** [comment]
Hey @shagupta92042,

work on this branch is not finished yet, according to @PiotrGackowski , so I'm closing the PR. 

Thanks,
Ludwik Kaźmierczak,
Wikia QM Tech Lead

**Pull request closed without merging**